### DISCUSSION TITLE
When i apply with converted files i get rejected.

### DISCUSSION THREAD

**Discussion by EmircanLalecan** [community/community#140666] [GitHub Education]
### Hi everyone,

Hi today i wanted to try github student pack benefits but every time i try to apply it rejects my application. In my country they provide us with pdf files but github wants png or jpeg. so i had to convert them into that. when i convert it recejts. when i don't convert i can't even submit. i guess i need some help here.

**Comment by Anit-K-Peter** [comment]
Try printing a copy of the document, then export the image as a PNG or JPEG and upload it; that might work.

**Reply by EmircanLalecan** [reply]
> Try printing a copy of the document, then export the image as a PNG or JPEG and upload it; that might work.

when i hit print key, there is no option like that

**Reply by Anit-K-Peter** [reply]
reach out github support to ensure any problems based on this

**Comment by Sanidhyafeaturist** [comment]
Make sure your PNG or JPEG files are saved correctly and clear. Double-check that you’re submitting all required documents. If you’re still having trouble, reach out to GitHub support or ask your school for help with the format. Good luck!

**Reply by EmircanLalecan** [reply]
> Make sure your PNG or JPEG files are saved correctly and clear. Double-check that you’re submitting all required documents. If you’re still having trouble, reach out to GitHub support or ask your school for help with the format. Good luck!

github support does not provide manual confirmations  and school can't give me this files as image because the system is not that great. so i'm stuck with what i have. need to try untill i got accepted.

**Reply by Sanidhyafeaturist** [reply]
It's unfortunate that GitHub support doesn't provide manual confirmations and your school can't give you the files in image format. I suggest trying different conversion tools or using image editing software to take screenshots of your documents. Keep submitting until something works, and document your attempts for future reference. Stay persistent!

**Reply by EmircanLalecan** [reply]
i just gave up on github. sadly a huge company like githup can't accept more variety of documents and doesn't let you convert them into what they want.

"The image you used appears to be edited. Manipulated images may not be used. Please try a different image."
reaction to the converted files

**Reply by Sanidhyafeaturist** [reply]
seems the image you used may violate GitHub Education's guidelines on manipulated images. To resolve this, please select an original or appropriately licensed image from sources like Unsplash or Pexels, or create your own to ensure compliance.

**Reply by Sanidhyafeaturist** [reply]
don't give up on github yet, I think you can find a good solution :) best of luck!

**Comment by lottopiedev** [comment]
Take an screenshot in clear content and rename and try uploading it
